schemas/com.snowplowanalytics.snowplow/client_session/jsonschema/1-0-2

```diff
@@ -0,0 +1,81 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a client-generated user session",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "client_session",
+		"format": "jsonschema",
+		"version": "1-0-2"
+	},
+	"type": "object",
+	"properties": {
+		"userId": {
+			"type": "string",
+			"format": "uuid",
```

Comment:
Hey @matus-tomlein - I am a bit worried about this format change as 1-0-2 uses regex instead.  According to @istreeter:

```
For Redshift we treat uuid as fixed-length char https://github.com/snowplow/schema-ddl/blob/master/modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/generators/TypeSuggestions.scala#L135-L136

whereas a string matching a pattern is a varchar.

[TypeSuggestions.scala](https://github.com/snowplow/schema-ddl/blob/master/modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/generators/TypeSuggestions.scala)
      case (Some(types), Some(Format.UuidFormat)) if types.possiblyWithNull(Type.String) =>
        Some(RedshiftChar(36))

The migration might be safe, but I certainly cannot say confidently.
```

Would it be possible to revert this format change to the previous type and save it for a v2 schema?

Reply:
Didn't realize that could be a problem, fixed!